source/nodejs/adaptivecards/src/card-elements.ts

```diff
@@ -1834,7 +1819,22 @@ export class Media extends CardElement {
             playButtonContainer.style.height = "100%";
             playButtonContainer.style.display = "flex";
             playButtonContainer.style.justifyContent = "center";
-            playButtonContainer.style.alignItems = "center";
+	     playButtonContainer.style.alignItems = "center";
+	     playButtonContainer.onclick = (e) => {
+		if (this.hostConfig.media.allowInlinePlayback) {
+		    let mediaPlayerElement = this.renderMediaPlayer();
+
+		    this.renderedElement.innerHTML = "";
+		    this.renderedElement.appendChild(mediaPlayerElement);
+
+		    mediaPlayerElement.play();
+		}
+		else {
+		    if (Media.onPlay) {
+			Media.onPlay(this);
+		    }
+		}
```

Comment:
Your indentation is off here -- can you take a look ? Make sure you have enabled viewing whitespace changes in  your editor/settings.